src/Http/Controllers/RequestController.php

```diff
@@ -9,15 +9,27 @@
 class RequestController extends Controller
 {
     /**
-     * Stores action and visit counts
+     * Saves action/visit count from cron job
      *
-     * @param StoreRequest $request containing the json object
+     * @api {post} /kpis
+     * @apiPermission token-verifyed
+     * @apiDescription Saves actions/visits of a serverlog file
      *
-    */
+     * @apiParamExample {JSON} Request example (JSON):
+     * {
+     *    {
+     *       "visits": 10,
+     *       "actions": 20
+     *    }
+     * }
+     *
+     *
+     * @param StoreRequest $request
+     */
     public function store(StoreRequest $request)
     {
-        $res = json_decode($request->get('value'), true);
-        Requests::save($res['actions'], $res['visits']);
-        return;
+        $visits = $visits = $request->input('visits');
+        $actions = $visits = $request->input('actions');
```

Comment:
This (visits=actions) should have been caught by the tests. please update the tests so they check the actual values in the database..

```suggestion
        $visits = $request->input('visits');
        $actions = $request->input('actions');
```